### PR TITLE
Starting collection of high quality comedy

### PR DESCRIPTION
**Pull request by mattiasgrenfeldt**
See changes for more details.

### CODE REVIEW

File: funny.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I don’t trust stairs. They’re always up to something.
+
```
Comment (kele) [inline funny.txt]: Please remove the unnecessary newline. Our linters would complain.

Comment (mattiasgrenfeldt) [reply funny.txt]: Fixed!

File: funny.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+I don’t trust stairs. They’re always up to something.
```
Comment (empijei) [inline funny.txt:1]: can you please add an explanation why the joke is funny?

Comment (mattiasgrenfeldt) [reply funny.txt:1]: Done in commit 2f07b12 "Explanation".

**Review by empijei — APPROVED** [review]

**Pull request merged**